README.md

```diff
@@ -151,8 +151,16 @@ To see the full set of available params, see [config/defaults.conf](config/defau
 
 To use the ON-LSTM sentence encoder from [Ordered Neurons: Integrating Tree Structures into Recurrent Neural Networks](https://arxiv.org/abs/1810.09536), set ``sent_enc = onlstm``. To re-run experiments from the paper on WSJ Language Modeling, use the configuration file [config/onlstm.conf](config/onlstm.conf). Specific ON-LSTM modules use code from the [Github](https://github.com/yikangshen/Ordered-Neurons) implementation of the paper.
 
-To use the PRPN sentence encoder from [***Neural language modeling by jointly learning syntax and lexicon***](https://arxiv.org/abs/1711.02013), set ``sent_enc=prpn``. To re-run experiments from the paper on WSJ Language Modeling, use the configuration file [config/prpn.conf](config/prpn.conf). Specific PRPN modules use code from the [Github](https://github.com/yikangshen/PRPN) implementation of the paper. 
- 
+To use the PRPN sentence encoder from [***Neural language modeling by jointly learning syntax and lexicon***](https://arxiv.org/abs/1711.02013), set ``sent_enc=prpn``. To re-run experiments from the paper on WSJ Language Modeling, use the configuration file [config/prpn.conf](config/prpn.conf). Specific PRPN modules use code from the [Github](https://github.com/yikangshen/PRPN) implementation of the paper.
+
+## Currently Supported Task Types
+We currently support the following:
+	* Single sentence classification tasks
+	* Pair sentence classification tasks
+	* Span Classification Tasks - to run these, we currently require an extra preprocessing 
+		step, which consists of preprocessing the data to get BERT tokenized span indice. 
+		SpanTasks expects the files to be in json format and be named as {file_name}.retokenized.{tokenizer_name}
```

Comment:
- SST is a regression task.
- Add a note about seq2seq tasks. Those are at least partially supported, so maybe add a note that support is limited/provisional.
- Look through the set of tasks for anything else worth mentioning here.